plugins/outputs/cloudwatchlogs/pusher.go

```diff
@@ -413,6 +413,9 @@ func (p *pusher) convertEvent(e logs.LogEvent) *cloudwatchlogs.InputLogEvent {
 			// a valid timestamp and use the last valid timestamp for new entries that does
 			// not have a timestamp.
 			t = p.lastValidTime
+			if ((time.Now().UnixNano() / 1000000000) - (t / 1000)) > 432000 {
```

Comment:
What are these magic numbers?

Reply:
For calculating the days, I convert the nanosecond value and divide by 1000000000 to convert into seconds. `t`is going to be in miliseconds so divide that by 1000

`432000` is the second value of 5 days so I'm trying to log the message when `lastValidTime` is 5 days from now

Reply:
Cleaned up check on time to use time.Hour for comparison and go builtin time.Since()